firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/internal/common/CrashlyticsController.java

```diff
@@ -628,13 +631,22 @@ void saveVersionControlInfo() {
   }
 
   String getVersionControlInfo() throws IOException {
+    // Attempt to read from an Android string resource
+    String versionControlInfo = CommonUtils.getVersionControlInfo(context);
+    if (versionControlInfo != null) {
+      Logger.getLogger().d("Read version control info from string resource");
+      return Base64.encodeToString(versionControlInfo.getBytes(UTF_8), 0);
+    }
+
+    // Fallback to reading the file
     InputStream is = getResourceAsStream(META_INF_FOLDER + VERSION_CONTROL_INFO_FILE);
```

Comment:
`InputStream is = getResourceAsStream(META_INF_FOLDER + VERSION_CONTROL_INFO_FILE);`
Should this in try block?

Reply:
Done